internal/services/compute/client/client.go

```diff
@@ -55,11 +58,11 @@ type Client struct {
 	VirtualMachinesClient            *virtualmachines.VirtualMachinesClient
 	VMExtensionImageClient           *compute.VirtualMachineExtensionImagesClient
 	VMExtensionClient                *compute.VirtualMachineExtensionsClient
-	VMScaleSetClient                 *compute.VirtualMachineScaleSetsClient
+	VMScaleSetClient                 *virtualmachinescalesets.VirtualMachineScaleSetsClient
 	VMScaleSetExtensionsClient       *compute.VirtualMachineScaleSetExtensionsClient
-	VMScaleSetRollingUpgradesClient  *compute.VirtualMachineScaleSetRollingUpgradesClient
+	VMScaleSetRollingUpgradesClient  *virtualmachinescalesetrollingupgrades.VirtualMachineScaleSetRollingUpgradesClient
 	VMScaleSetVMsClient              *compute.VirtualMachineScaleSetVMsClient
-	VMClient                         *compute.VirtualMachinesClient
+	VMClient                         *virtual_machines_2023_03_01.VirtualMachinesClient
```

Comment:
could we update the version used by `VirtualMachinesClient` and use that instead?

Reply:
Good shout! Done!